### PR TITLE
Add Deploy Operation and Maintenance FAQs

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)

- Refactor the FAQ documentation related to operation and maintenance for deployment. > [Deploy Operation and Maintenance FAQs ](https://docs.pingcap.com/zh/tidb/dev/deploy-and-maintain-faq)

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3268
- Other reference link(s):

### CODE REVIEW

File: faq/deploy-and-maintain-faq.md [hunk 1]
```diff
@@ -0,0 +1,548 @@
+---
+title: Deploy and Maintain FAQ
+summary: Introduce FAQs, causes and solutions relating to TiDB cluster operation and maintenance deployment.
+category: FAQ
```
Comment (TomShawn) [inline faq/deploy-and-maintain-faq.md:4]:
```suggestion
```

File: faq/deploy-and-maintain-faq.md [hunk 2]
```diff
@@ -0,0 +1,549 @@
...
+
+At the beginning, many users tend to do a benchmark test or a comparison test between TiDB and MySQL. We have also done a similar official test and find the test result is consistent at large, although the test data has some bias. Because the architecture of TiDB differs greatly from MySQL, it is hard to find a benchmark point. The suggestions are as follows:
+
+- Do not spend too much time on the benchmark test. Pay more attention to the difference of scenarios using TiDB.
+- See [Performance test result for TiDB using Sysbench](/benchmark/v3.0-performance-benchmarking-with-sysbench.md).
+
+#### What's the relationship between the TiDB cluster capacity (QPS) and the number of nodes? How does TiDB compare to MySQL?
+
+- Within 10 nodes, the relationship between TiDB write capacity (Insert TPS) and the number of nodes is roughly 40% linear increase. Because MySQL uses single-node write, its write capacity cannot be scaled.
+- In MySQL, the read capacity can be increased by adding slave, but the write capacity cannot be increased except using sharding, which has many problems.
```
Comment (Joyinqin) [inline faq/deploy-and-maintain-faq.md:501]: 待确定敏感词

File: faq/deploy-and-maintain-faq.md [hunk 3]
```diff
@@ -0,0 +1,549 @@
...
+
+At the beginning, many users tend to do a benchmark test or a comparison test between TiDB and MySQL. We have also done a similar official test and find the test result is consistent at large, although the test data has some bias. Because the architecture of TiDB differs greatly from MySQL, it is hard to find a benchmark point. The suggestions are as follows:
+
+- Do not spend too much time on the benchmark test. Pay more attention to the difference of scenarios using TiDB.
+- See [Performance test result for TiDB using Sysbench](/benchmark/v3.0-performance-benchmarking-with-sysbench.md).
+
+#### What's the relationship between the TiDB cluster capacity (QPS) and the number of nodes? How does TiDB compare to MySQL?
+
+- Within 10 nodes, the relationship between TiDB write capacity (Insert TPS) and the number of nodes is roughly 40% linear increase. Because MySQL uses single-node write, its write capacity cannot be scaled.
+- In MySQL, the read capacity can be increased by adding secondary, but the write capacity cannot be increased except using sharding, which has many problems.
```
Comment (Joyinqin) [inline faq/deploy-and-maintain-faq.md:501]: “具体参考方案虽好，成本先行：数据库 Sharding+Proxy 实践解析“是中文文章 英文这里省略掉了。

File: faq/deploy-and-maintain-faq.md [hunk 4]
```diff
@@ -0,0 +1,549 @@
...
+
+If you previously deploy a PD cluster, but then you remove the PD data and deploy a new PD cluster, this error occurs because TiKV uses the old data to connect to the new PD cluster.
+
+#### The `duplicated store address` message is displayed when starting TiKV
+
+This is because the address in the startup parameter has been registered in the PD cluster by other TiKVs. Common conditions that cause this error: There is no data folder in the path specified by TiKV `--data-dir` (no update --data-dir after deleting or moving), restart the TiKV with the previous parameters.Please try [store delete](https://github.com/pingcap/pd/tree/55db505e8f35e8ab4e00efd202beb27a8ecc40fb/tools/pd-ctl#store-delete--label--weight-store_id----jqquery-string) function of pd-ctl, delete the previous store, and then restart TiKV.
+
+#### TiKV primary node and secondary node use the same compression algorithm, why the results are different?
```
Comment (Joyinqin) [inline faq/deploy-and-maintain-faq.md:392]: Not sure whether 'node' can be used here.

File: faq/deploy-and-maintain-faq.md (hunk 3 above)
Comment (lilin90) [reply faq/deploy-and-maintain-faq.md]: OK.

**Pull request merged**